README.md

```diff
@@ -45,7 +45,7 @@ Using jupyter, run the default `training.ipynb` notebook. All adjustable paramet
 The `Transformer` class can be used out of the box, [docs](https://timeseriestransformer.readthedocs.io/en/latest/Transformer.html) for more info.
 
 ```python
-from tst.Transformer import Transformer
+from test.Transformer import Transformer
```

Comment:
I don't understand why you would change tst (Time Series Transformer), the name of the package, with test.

Reply:
My bad! I thought that was supposed to be test folder. It is a good practice to always have a test folder in every project so that key features are always tested in order to increase robustness.